cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -52,15 +65,70 @@ def test_myservice_crud(appliance, setup_provider, context, order_catalog_item_i
         my_service.delete()
 
 
-@pytest.mark.uncollectif(lambda: current_version() < '5.8')
+@pytest.mark.uncollectif(lambda: current_version() < '5.8' or current_version() > '5.9')
 @pytest.mark.parametrize('context', [ViaSSUI])
 @pytest.mark.parametrize('order_catalog_item_in_ops_ui', [['console_test']], indirect=True)
-def test_vm_console(appliance, setup_provider, context, configure_websocket,
-        order_catalog_item_in_ops_ui):
+def test_vm_console(request, appliance, setup_provider, context, configure_websocket,
+        order_catalog_item_in_ops_ui, take_screenshot, console_template):
     """Test Myservice VM Console in SSUI."""
     catalog_item = order_catalog_item_in_ops_ui
     service_name = catalog_item.name
+    console_vm_username = credentials[catalog_item.provider.data.templates.get('console_template')
```

Comment:
**Required**: console_vm_username = credentials[catalog_item.provider.data.templates.console_template.creds].username
As we can access conf values as attr
Please do the same for password